### ISSUE TITLE
Static files are not loaded in IE8 on XP.

### ISSUE THREAD

**Issue by djallado** [learningequality/installers#176]
**Branch:** `0.15.x` 

**Windows installer:**  `kalite-setup-0.15.0--9-18-2015.exe`

**Platform:** `IE8 on XP`

**Expected Behavior:** Must load static files.

**Current Behavior:** Static files are not loaded.

**Steps to reproduce:**
 1. Download windows installer at http://field.learningequality.org/KALiteSetup-0.15.0--9-18-2015.exe.
 2. Install it in IE8 on XP.
 3. Launch KA-Lite app
 4. Click on `Start on Server` in menu option.
 5. Wait until the `Load in browser` menu becomes available and click on it.

**Screenshot:** 
 
![staticfile](https://cloud.githubusercontent.com/assets/8664071/10064127/d904b4a4-62a7-11e5-905d-82b90b812eb7.png)

**Comment by MCGallaspy** [comment]
Not sure what the issue could be here. I'll try this out. Does it load the staticfiles if you restart the machine?

**Comment by MCGallaspy** [comment]
Also curious -- did you restart the machine after completing setup? You should have been prompted to restart.

**Comment by djallado** [comment]
The machine was restarted after I clicked the finish button. I' using virtual machine with fresh install IE8 on XP.

**Comment by MCGallaspy** [comment]
Trying to replicate.

**Comment by MCGallaspy** [comment]
This is an issue with IE8 -- I was able to replicate it quite easily by just installing normally, as you described. On the same XP machine, I tried it out in firefox -- everything works great. Since we don't support IE8 anyway, I'm closing this issue. Windows XP users will have to use a modern browser.

Probably due to our use of jquery2 -- [it's not supported on IE8](https://jquery.com/browser-support/).
